sidecar/main.go

```diff
@@ -0,0 +1,49 @@
+/*
+Copyright 2021 The Kubernetes-CSI-Addons Authors.
+
+Licensed under the Apache License, Version 2.0 (the "License");
+you may not use this file except in compliance with the License.
+You may obtain a copy of the License at
+
+    http://www.apache.org/licenses/LICENSE-2.0
+
+Unless required by applicable law or agreed to in writing, software
+distributed under the License is distributed on an "AS IS" BASIS,
+WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+See the License for the specific language governing permissions and
+limitations under the License.
+*/
+
+package main
+
+import (
+	"flag"
+	"time"
+
+	"github.com/csi-addons/kubernetes-csi-addons/sidecar/internal/client"
+	"k8s.io/klog/v2"
+)
+
+func main() {
+	var (
+		defaultTimeout   = time.Minute * 3
+		timeout          = flag.Duration("timeout", defaultTimeout, "Timeout for waiting for response")
+		csiAddonsAddress = flag.String("csi-addons-address", "/run/csi-addons/socket", "CSI Addons endopoint")
```

Comment:
Maybe use the same default as what we added for Ceph-CSI? Unless this matches what other standard Kubernetes CSI  containers use, in that case it is fine (just confirm here).

Reply:
external provisioner and snapshotter use `csi-address` .

https://github.com/kubernetes-csi/external-snapshotter/blob/138d310e5d2d102fcf90df96d7a8aaf6690ce76e/cmd/csi-snapshotter/main.go#L60
https://github.com/kubernetes-csi/external-snapshotter/blob/138d310e5d2d102fcf90df96d7a8aaf6690ce76e/cmd/csi-snapshotter/main.go#L60
 
:sweat_smile: I did modify it couple of times while developing, either name is fine for me.
Let me know which you think we should use.